src/core/server/http/http_server.test.ts

```diff
@@ -1166,7 +1166,7 @@ describe('timeout options', () => {
       router.get(
         {
           path: '/',
-          validate: { body: schema.any() },
+          validate: false, // TODO: Figure out if we can/should use `{ body: schema.any() }` instead
```

Comment:
@elastic/kibana-core without this change to the test, the test-server would return:

```js
    {
      statusCode: 400,
      error: 'Bad Request',
      message: '[request body]: expected value of type [any] but got [undefined]'
    }
```

I'm not sure exactly what changed between hapi 18 and 20 that caused this, so I'm not super comfortable with this change to the test. But I can't figure out exactly why it's behaving differently, so I would appreciate another pair of eyes.

Reply:
IIRC, Hapi used to default `request.payload` to an empty object, even if no body was provided. I believe `schema.any` uses `Joi.any` under the hood which does not accept `undefined` by default. If we want to avoid changing this, we could change the value passed into the validator here to default to an empty object: https://github.com/elastic/kibana/blob/master/src/core/server/http/router/request.ts#L144

That said, I actually prefer `undefined` as the default because it maps better to what I would expect from an empty HTTP payload. It allows the application to distinguish between an empty body and an actual `{}` JSON string. If others agree, we could change this test code to `validate: schema.maybe(schema.any())`.

Reply:
Fixed by wrapping in `schema.maybe` as suggested 👍